lib/PipelineStack.ts

```diff
@@ -5,6 +5,8 @@ import * as codePipeline from "aws-cdk-lib/aws-codepipeline";
 import * as codeCommit from 'aws-cdk-lib/aws-codecommit';
 import * as codePipelineActions from "aws-cdk-lib/aws-codepipeline-actions";
 import * as codeCommitActions from "aws-cdk-lib/aws-codepipeline-actions";
+import * as secrets from "aws-cdk-lib/aws-secretsmanager";
+import {GitHubTrigger} from "aws-cdk-lib/aws-codepipeline-actions";
```

Comment:
Can you match the import style please for consistancy?

Reply:
Good catch. Will fix.